YouTubeVideoSearch/src/actions/actions.js

```diff
@@ -0,0 +1,24 @@
+import { fetchYoutubeVideo } from "../service/service";
+
+export const fetchVideo = searchText => async dispatch => {
+  const searchResult = await fetchYoutubeVideo(searchText);
+  let searchResultJson = await searchResult.json();
+  let videos = [];
+  if (searchResultJson.items) {
+    let items = searchResultJson.items;
+    items.forEach(function(item) {
+      let video = {};
+      video.id = item.id.videoId;
```

Comment:
I would recommend the following syntax:
const video = {
id: item.id.videoId
...
};